src/panels/config/logs/error-log-card.ts

```diff
@@ -51,11 +59,20 @@ class ErrorLogCard extends LitElement {
                   <h2>
                     ${this.hass.localize("ui.panel.config.logs.full_logs")}
                   </h2>
-                  <ha-icon-button
-                    .path=${mdiRefresh}
-                    @click=${this._refresh}
-                    .label=${this.hass.localize("ui.common.refresh")}
-                  ></ha-icon-button>
+                  <div>
+                    <ha-icon-button
```

Comment:
"Show full logs" is considered the primary action here right? shouldn't that be on the right side?

Reply:
Adjusted

<img width="1006" alt="Screenshot 2022-11-28 at 4 09 11 PM" src="https://user-images.githubusercontent.com/663432/204421213-af66f770-8470-42a3-9614-b6a4696b5f34.png">